torchaudio/backend/utils.py

```diff
@@ -33,6 +35,16 @@ def list_audio_backends() -> List[str]:
     return backends
 
 
+_USE_SOUNDFILE_LEGACY_INTERFACE = True
```

Comment:
Alternatively (and probably saver) you can do
```
torchaudio._USE_SOUNDFILE_LEGACY_INTERFACE=True
```
and create a _USE_SOUNDFILE_LEGACY_INTERFACE in the ```torchaudio/__init__.py``` file.

Reply:
What about someone doing

```
def fn1():
    import torchaudio
    torchaudio.backend.utils.use_soundfile_legacy_interface(True)
    def _fn():
         torchaudio.set_backend('soundfile')
         torchaudio.<do_something>

def fn2(
    import torchaudio
    torchaudio.backend.utils.use_soundfile_legacy_interface(False)

fn = fn1()
fn2()
fn() # Is this using the legacy or non-legacy version?
```

Maybe this is fixed if instead of using the global, you create a variable in ```torchaudio/__init__.py```?

Reply:
Okay, so it seems that accessing the variable directly has more advantage (like users can check the current configuration), so I will move to it.

Reply:
Moved "USE_SOUNDFILE_LEGACY_INTERFACE" to `torchaudio.USE_SOUNDFILE_LEGACY_INTERFACE` and got rid of the helper function.